ethportal-api/src/types/cli.rs

```diff
@@ -207,13 +219,25 @@ impl TrinConfig {
         }
 
         match config.web3_transport {
-            Web3TransportType::HTTP => match &config.web3_ipc_path.as_path().display().to_string()[..] {
-                DEFAULT_WEB3_IPC_PATH => {}
-                _ => return Err(Error::raw(ErrorKind::ArgumentConflict, "Must not supply an ipc path when using http protocol for json-rpc")),
-            },
-            Web3TransportType::IPC => match config.web3_http_address.as_str() {
-                DEFAULT_WEB3_HTTP_ADDRESS => {}
-                p => return Err(Error::raw(ErrorKind::ArgumentConflict,format!("Must not supply an http address when using ipc protocol for json-rpc (received: {p})"))),
+            Web3TransportType::HTTP => {
+                match &config.web3_ipc_path.as_path().display().to_string()[..] {
+                    DEFAULT_WEB3_IPC_PATH => {}
+                    _ => {
+                        return Err(Error::raw(
+                            ErrorKind::ArgumentConflict,
+                            "Must not supply an ipc path when using http protocol for json-rpc",
+                        ))
+                    }
+                }
+            }
+            Web3TransportType::IPC => {
+                match config.web3_http_address.as_str() {
+                    DEFAULT_WEB3_HTTP_ADDRESS => {}
+                    p => return Err(Error::raw(ErrorKind::ArgumentConflict,format!("Must not supply an http address when using ipc protocol for json-rpc (received: {p})"))),
```

Comment:
I know it's not your change, but can you update this single letter variable to something more descriptive